xml/admin_monitoring_alerting.xml

```diff
@@ -1686,27 +1675,27 @@ ceph-volume lvm create --osd-id $N --data /path/to/device</screen>
       <listitem>
         <para>The number of PGs in use in the cluster is above the configurable
           threshold of <option>mon_max_pg_per_osd</option> PGs per OSD. If this threshold is
-          exceeded, the cluster will not allow new pools to be created, pool <option>pg_num</option>
+          exceeded, the cluster does not allow new pools to be created, pool <option>pg_num</option>
           to be increased, or pool replication to be increased (any of which
           would lead to more PGs in the cluster). A large number of PGs can
           lead to higher memory utilization for OSD daemons, slower peering
           after cluster state changes (like OSD restarts, additions, or
           removals), and higher load on the &mgr; and &mon; daemons.
           The simplest way to mitigate the problem is to increase the number
-          of OSDs in the cluster by adding more hardware. Note that the OSD
+          of OSDs in the cluster by adding more hardware. The OSD
           count used for the purposes of this health check is the number
-          of “in” OSDs, so marking “out” OSDs “in” (if there are any) can also help:</para>
+          of <option>in</option> OSDs, marking <option>out</option> OSDs <option>in</option> (if there are any) can also help:</para>
```

Comment:
This line is so long I had to scroll to read it.

Reply:
The outrage ;)

Reply:
AFAICS it's not in a [screen] tag or anything so it could be line-wrapped...?

Reply:
yes, i vote for auto xmlformat :-)